Storage/storage.h

```diff
@@ -75,26 +75,6 @@ namespace storage {
 // 2) Splits the type into `T_ADDER` and `T_DELETER`, to support seamless persistence of deletions.
 // clang-format off
 
-#ifndef _MSC_VER
```

Comment:
Yes. I'm just reluctant to merge multiple changes in knowing there will be
more changes, and I won't see Windows for the next two days. Fixing it back
later, even if it's a one line change, would be a pain.